app/src/main/java/app/revanced/manager/domain/manager/PreferencesManager.kt

```diff
@@ -9,6 +9,7 @@ class PreferencesManager(
 ) : BasePreferencesManager(context, "settings") {
     val dynamicColor = booleanPreference("dynamic_color", true)
     val theme = enumPreference("theme", Theme.SYSTEM)
+    val showManagerUpdateDialog = booleanPreference("show_manager_update_dialog", true)
```

Comment:
Consider moving this next to `managerAutoUpdates` (also, doesn't this serve the exact same purpose as that preference?).

Reply:
Do you want Manager to not check for ReVanced Manager updates when dismissed with never show again? I believe we could still show the Update available badge in the TopAppBar actions

Reply:
I marked this as resolved by mistake

Reply:
```
BooleanItem(
    preference = vm.managerAutoUpdates,
    headline = R.string.update_checking_manager,
    description = R.string.update_checking_manager_description
)
```

Doesn't really allow me to do that